wiki/es/es-Running-a-Server.md

```diff
@@ -1,16 +1,15 @@
 ---
+lang: es
 layout: wiki
-title: "Ejecutar un Servidor"
-lang: "es"
-permalink: "/wiki/Running-a-Server"
+permalink: /wiki/Running-a-Server
+title: 'Ejecutar un Servidor'
 ---
 
 # Ejecutar un Servidor
 
 ## ¿Necesito ejecutar un servidor para utilizar Jamulus?
 
-NO.
-{: .doubletextsize .red }
+NO. {: .doubletextsize .red }
```

Comment:
Are you sure this works with jekyll?

Reply:
Well spotted. It actually has to be:

NO.
 {: .doubletextsize .red }

With a space at the start of the second line for it to work properly with po4a.